src/_base.scss

```diff
@@ -9,7 +9,6 @@ html {
 body {
     width: 100vw;
     height: 100vh;
-    color: #333;
-    background: url('https://static.pexels.com/photos/167672/pexels-photo-167672.jpeg') center/cover;
+    color: black;
```

Comment:
`color: #333` is better than black. Golden rule: don't ever use the blackest from the black.